FirebaseAdmin/FirebaseAdmin/Auth/UserImportResult.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+
+namespace FirebaseAdmin.Auth
+{
+    /// <summary>
+    /// Represents the result of the
+    /// <a cref="o:FirebaseAuth.ImportUsersAsync">FirebaseAuth.ImportUsersAsync</a> API.
+    /// </summary>
+    public sealed class UserImportResult
+    {
+        private readonly int users;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UserImportResult"/> class based on supplied
+        /// users and <a cref="FirebaseUserManager.UploadAccountResponse">UploadAccountResponse</a> objects.
+        /// </summary>
+        /// <param name="users"> The number of users.</param>
+        /// <param name="response"> The UploadAccountResponse generated from the post request.</param>
+        internal UserImportResult(int users, FirebaseUserManager.UploadAccountResponse response)
```

Comment:
Perhaps this constructor can accept `IEnumerable<ErrorInfo>` instead of `UploadAccountResponse`?